audio_extraction/audio_extraction.ipynb

```diff
@@ -0,0 +1,245 @@
+    "                    encoding = 'utf-8', level = log_switch_case[log_flag],\n",
+    "                    filemode = 'w')\n",
+    "    \n",
+    "    # Making a default directory for to_folder\n",
+    "    if to_folder == None:\n",
+    "        print(\"Created a default to_folder as none was given.\")\n",
+    "        video_path.parent.joinpath('extracted_audio')\n",
+    "    \n",
+    "    # Create a counter for the number of audio files processed\n",
+    "    audio_file_counter = 0\n",
+    "    \n",
+    "    # Go through all files in `video_path`\n",
+    "    for child in video_path.iterdir():\n",
+    "        # Debugging: show the file name\n",
+    "        logging.debug('Detected file: {}'.format(child.name))\n",
+    "        # Store the extension of the file name\n",
+    "        file_extension = Path(child.name).suffix\n",
+    "        # If the file's extension is in `video_extensions`\n",
+    "        if file_extension in video_extensions:\n",
+    "            # Do an ffmpeg probe on the currently selected file\n",
+    "            current_probe = ffmpeg.probe(child)\n",
+    "            # If the number of streams is more than 1\n",
+    "            # (as a proxy for if the file could have audio)\n",
+    "            if len(current_probe['streams']) > 1:\n",
+    "                # Check if `codec_type` is audio\n",
+    "                for stream in current_probe['streams']:\n",
+    "                    # Debugging: print the codec type\n",
+    "                    logging.debug('Codec type: {}'.format(stream['codec_type']))\n",
+    "                    # If the codec type is audio\n",
+    "                    if stream['codec_type'] == 'audio':\n",
+    "                        # Debugging: print the video file's path\n",
+    "                        logging.debug('Audio codec detected: {}'.format(child))\n",
+    "                        # Store the video file path\n",
+    "                        new_child = str(child)\n",
+    "                        # Replace the extension with .mp3\n",
+    "                        new_child = re.sub(pattern = '{}$'.format(file_extension),\n",
+    "                                          repl = '.mp3', string = new_child)\n",
+    "                        \n",
+    "                        # Check that the audio path exists\n",
+    "                        if not a_path_1.exists():\n",
+    "                            print(\"to_folder directory does not exist therefore one will be made.\")\n",
+    "                            a_path_1.mkdir(parents = True, exist_ok = False)\n",
+    "                        \n",
+    "                        # Create a variable to store the audio file's path\n",
+    "                        audio_file_path = a_path_1.joinpath(Path(new_child).name)\n",
+    "                        # Debugging: print the audio file's path\n",
+    "                        logging.debug('Audio file extracted to: {}'.format(audio_file_path))\n",
```

Comment:
This needs to come after the try, and only if it succeeds, you don't want this to print if it does not actually extract

Reply:
So just put it in the try after the extraction

Reply:
I've moved this code into the try block:

```
try:
    error_message = ffmpeg.input(str(child)) \
    .output(filename = str(audio_file_path),
        **{'qscale:a': 0,
        'map': 'a'}) \
        .overwrite_output() \
        .run()

    # Create a variable to store the audio file's path
    audio_file_path = Path(to_folder).joinpath(
        Path(new_child).name
    )
    # Debugging: print the audio file's path
    logging.debug('Audio file extracted to: ' +
                             f'{audio_file_path}')
```